src/components/NurseryWithdrawals/PlantingProgressTabContent.tsx

```diff
@@ -117,10 +117,8 @@ type SearchComponentProps = SearchProps & {
 function SearchComponent(props: SearchComponentProps): JSX.Element {
   const { search, onSearch, filtersProps, view } = props;
   return (
-    <>
-      <div style={{ display: view === 'list' ? 'flex' : 'none' }}>
-        <Search search={search} onSearch={onSearch} filtersProps={filtersProps} />
-      </div>
-    </>
+    <div style={{ display: 'flex' }}>
```

Comment:
need a non-zero div for `justify-content: space-between` style to work correctly.. otherwise the list/map view selector was on the left hand side